frontend/src/Home.jsx

```diff
@@ -1,13 +1,12 @@
 import React from "react";
-import logo from "./logo.png";
+import big_welcome_banner from "./images/big_welcome_banner.png";
 import LOCALIZE from "./text_resources";
 
 const Home = () => {
   return (
     <div className="app">
-      <h2>{LOCALIZE.homePage.title}</h2>
-      <p>{LOCALIZE.homePage.welcomeMsg}</p>
-      <img src={logo} alt="logo" />
+      <h2 id="home-page-paragraph">{LOCALIZE.homePage.title}</h2>
```

Comment:
Since we don't have an H1 on the page yet, I think this needs to be an h1

Reply:
resolved in eba175e